kernel/base/src/main/java/com/twosigma/beakerx/evaluator/BaseEvaluator.java

```diff
@@ -27,10 +32,27 @@
   protected abstract boolean removeImportPath(ImportPath anImport);
 
   @Override
-  public void addJarToClasspath(PathToJar path) {
-    if (addJar(path)) {
+  public boolean addJarToClasspath(PathToJar path) {
+    boolean added = addJar(path);
+    if (added) {
       resetEnvironment();
     }
+
+    return added;
+  }
+
+  @Override
+  public List<Path> addJarsToClasspath(List<PathToJar> paths) {
+    LinkedList<Path> addedPaths = Lists.newLinkedList();
+    paths.forEach(path -> {
+      if (addJar(path)) {
+        addedPaths.add(Paths.get(path.getPath()));
+      }
+    });
+
+    resetEnvironment();
```

Comment:
Should this be conditional on `!addedPaths.isEmpty()` for consistency with the single jar case?